src/examples/Examples.js

```diff
@@ -90,6 +92,25 @@ var Examples = React.createClass({
           justify="end" />
         <Box pad={{horizontal: 'large'}}><p><strong>Grommet-only Marquee (no parallax, responsiveness)</strong></p></Box>
         {this._loremIpsum()}
+        <Box pad={{ horizontal: 'large', vertical: 'none' }}>
+          <Callout
+            thumbnail="url(/docs/img/Video_image.png)"
```

Comment:
Can we make this so that the `thumbnail` prop will just take the actual path or URL to the image? This way, we are not forcing people to know how to formulate CSS background properties in order to use this component.

By the way, is there a doc page for this component?